pkg/cluster/internal/providers/crafting/OWNERS

```diff
@@ -0,0 +1,2 @@
+labels:
+- area/provider/docker
```

Comment:
What is this used for?

Reply:
It is only used for PR review. It doesn't affect code. I will update it to `area/provider/crafting`